.github/workflows/deploy-preproduction.yml

```diff
@@ -155,6 +179,17 @@ jobs:
             RELEASE_VERSION=${{ env.RELEASE_VERSION }}
           tags: ${{ secrets.PREPROD_REPOSITORY_BASE_PATH }}/runner-pulumi-javascript:${{ env.RELEASE_VERSION }}
 
+      - name: Build and push the fips image
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          platforms: linux/amd64,linux/arm64
+          context: javascript
+          build-args: |
+            REPOSITORY_BASE_PATH=${{ secrets.PREPROD_REPOSITORY_BASE_PATH }}
+            RELEASE_VERSION=${{ env.RELEASE_VERSION }}-fips
```

Comment:
Do we need base image here?

Reply:
and in a few of the ones that follow?

Reply:
I dont think so, for the specific language image we use a base alpine/debian image

Reply:
https://github.com/spacelift-io/runner-pulumi/actions/runs/13971817551/job/39115321840 you can check the logs from the build action